src/DynamoCoreWpf/Views/Search/LibrarySearchView.xaml

```diff
@@ -710,7 +710,7 @@
                                                     </TextBlock>
                                                     <ListBox Background="Transparent"
                                                              BorderBrush="Transparent"
-                                                             ItemContainerStyle="{DynamicResource ListBoxMembersStyle}"
+                                                             ItemContainerStyle="{StaticResource ListBoxMembersStyle}"
```

Comment:
If this fixes the output in debugger, then perhaps it can be made a standalone PR so it can be quickly merged in. Also, @ke-yu has reported another exception when we start debugging Dynamo (with .NET exceptions enabled), if you could fix this also in the same standalone PR, that will be really helpful:

    "Object reference not set to an instance of an object."
    at Controls\controls\HeaderStrip.xaml:line 85